helpers.h

```diff
@@ -2,16 +2,16 @@
 #include "bmp.h"
 
 // Convert Image to grayscale
-void grayscale(int height,int width,RGBTRIPLE **image);
+  void grayscale(int ,int ,RGBTRIPLE);
 
 // Reflect image horizontally
-void reflect(int height,int width,RGBTRIPLE **image);
+  void reflect(int ,int ,RGBTRIPLE);
```

Comment:
```suggestion
void reflect(int , int , RGBTRIPLE **);
```
Please, maintain what I have suggested.